hathor/p2p/netfilter/__init__.py

```diff
@@ -0,0 +1,5 @@
+from hathor.p2p.netfilter.instances import get_table
+
+__all__ = [
+    'get_table',
+]
```

Comment:
Should we add some simple tests? (This has nothing to do with this code location)

Reply:
Surely we should. I'll do it. Thanks!

Reply:
Done!